AVIenabledbyArc/vi-edge-deployment-script.sh

```diff
@@ -187,23 +240,31 @@ if [[ $install_aks_cluster == "true" ]]; then
       echo -e "\tAdding ingress controller -- ***start***"
       kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.7.0/deploy/static/provider/cloud/deploy.yaml
       echo -e "\tAdding ingress controller -- ***done***"
+      #=============================================#
+      #========= Patch Public IP DNS Label =========
+      #=============================================#    
+      publicIpResourceId=$(az network public-ip list --resource-group $nodePoolRg --query "[?contains(name, 'kubernetes')].id" -otsv)
+      echo "publicIpResourceId: $publicIpResourceId"
+      az network public-ip update --ids $publicIpResourceId --dns-name $dnsPrefix
+      echo "Public IP DNS Label has been updated to $dnsPrefix"
+      
       #=============================================#
       #============== Create AKS ARC Cluster =======#
       #=============================================#
-      echo -e "\tConneting AKS to ARC-AKS -- ***start***"
+      echo -e "\tConnecting AKS to ARC-AKS -- ***start***"
       az connectedk8s connect --name $connectedClusterName --resource-group $connectedClusterRg --yes --tags $tags
-      echo -e "\tconneting AKS to ARC-AKS -- ***done***"
+      echo -e "\tconnecting AKS to ARC-AKS -- ***done***"
 fi
 
 #===============================================================================#
 #====== Creating Cognitive Services on Behalf of the user on VI RP =============#
 #===============================================================================#
 create_cognitive_hobo_resources
```

Comment:
this does not install the extensino , this command creates the Connected cluster , which is 6-7 min infra operation.
if we move this part to the extension creation , it will run on each extension deploy , where in fact this is one-time infra operation.